packages/dataviews/src/types.ts

```diff
@@ -207,6 +215,7 @@ export type EditConfigGeneric = {
 export type EditConfig =
 	| EditConfigTextarea
 	| EditConfigText
+	| EditConfigNumber
```

Comment:
Do we need this now? I'd rather not introduce until when/if we need it.

Reply:
Oh, I forgot to update the type! Thanks for the headup!